homework5/scripts/task1.js

```diff
@@ -0,0 +1,28 @@
+/*
+შექმენით ფუნქცია, რომელიც იპოვის ყველზე გრძელ სიტყვას მომხმარებლის მიერ შეყვანილ სტრიქონში. (გამოიყენეთ prompt()-მონაცემების შესაყვანად.)
+*/
+
+let string = prompt("Please enter some words and I will return to you which of them is the longest");
+let stringToArray = string.split(" ");
+let main = [" "];
+let equalMain = [];
+
+
+let finder = stringToArray.forEach(function (index) {
+
+    if (index.length > main[0].length) {
+        main.push(index);
+        main.shift();
+    } else if (index.length = main[0].length){
+        equalMain.push(index);
+    };
+
+});
+
+
+let finder2 = equalMain.forEach(function (index) {
+    if (index.length === main[0].length) {
+        main.push(index);
+    }
+})
+alert("Longest word(s): " + main.join(' , '));
```

Comment:
ეს დავალება უფრო მოკლედ შეგეძლო დაგეწერა, აბა ეს კოდი ნახე:
let someString = prompt("Please enter some words and I will return to you which of them is the longest");
let arr = someString.split(" ");
let arrLen = arr.map(string=> string.length)
console.log("Longest word" + arr[arrLen.indexOf(Math.max(...arrLen))])
პატარა მოდიფიცირებით რამოდენიმე ერთი ზომის ყველაზე დიდ სიტყვასაც გამოიტანს ეს კოდი.

შენიც კარგია :) უბრალოდ ყოველთვის რაც ნაკლები ციკლები გექნება მით უკეთესია. და ამიტომაც ჩემსასაც გადახედე